### ISSUE TITLE
License Violation

### ISSUE THREAD

**Issue by anges244** [nextglabs/undraw-figma#1]
Hey Bassem,

Thank you for you eagerness to create an integration for unDraw on Figma but it's against our [license](https://undraw.co/license). Can you please unpublish the repo and the plugin from the Figma community?

**Comment by willisvt** [comment]
@anges244 This is a bummer.  I don't understand how it's different from the XD plugin.  Was exclusivity negotiated with XD?

**Comment by anges244** [comment]
Yes, we had discussed it at length and decided to proceed together to create that.

We maintain unDraw ourselves and Katerina (the creator) wants to keep it ad-free and open (no monetisation) so that's why we have to take calculated steps regarding how it expands and how many users we can support. That's why it's pretty clear even in the [landing page](https://undraw.co) that we don't want such integrations and mass downloads.

**Comment by nextglabs** [comment]
Hello @anges244 and thanks for reaching out.

I didn't mean to violate Undraw's license, which I have read beforehand which didn't _clearly_ mention this.
Anyway, I created this plugin because I'm a Figma user and I found out that this could help improve my workflow. This is why I decided to share it. There is no intention to create ads or monetization. 

How about we make it official within undraw? I'd be happy to help out in any way. I also believe that this would allow my work to help other people, rather than being tossed out in the garbage bin. 

We can talk about it in private if you'd like. I really hope that together, we can come up with a solution to this matter.

**Comment by anges244** [comment]
Hi, you really read the license and didn't _clearly_ get this?

In the landing page and on top of the license page: **open-source license**

You can use the illustrations in any project, commercial or personal without attribution or any costs. A truly open license, babe! **Just don’t try to replicate unDraw, re-distribute the illustrations or create integrations for it.**

In the license page, a special paragraph for that: **a rule of thumb (tldr)**

If you are working on something and want to use illustrations to improve its appearance, modified or not, without the need for attribution or cost, you are good to go. If you find unDraw or its illustrations to be in the center of what you are doing (e.g. sell or re-distribute one/some of them, **add them in an app), then you probably should not proceed.**

In the license text: 
This extends to automated and non-automated ways to **link**, **embed**, scrape, **search** or **download** the assets included on the website without our consent.

I can't think of how to make that more _"clear"_ as you say. The problem is, for reasons I just explained above, we don't want this integration. So I don't see how this can remain published. You should have checked with us before moving forward with this and I would have saved you the time. Hope this provides a bit more context on why you have to unpublish the repo and the plugin.
